products/azul-zulu.md

```diff
@@ -0,0 +1,211 @@
+    releaseLabel: "18 (<abbr title='Short Term Support'>STS</abbr>)"
+    lts: false
+    # https://docs.azul.com/core/zulu-openjdk/release-notes/18-ga
+    releaseDate: 2022-03-12
+    eol: 2022-09-30
+    extendedSupport: 2023-03-31
+    latest: "18.32.13"
+    latestJdkVersion: "18.0.2.1+1"
+    latestReleaseDate: 2022-08-22
+    link: https://docs.azul.com/core/zulu-openjdk/release-notes/11.58-17.36-august-2022
+
+-   releaseCycle: "17"
+    lts: true
+    # https://docs.azul.com/core/zulu-openjdk/release-notes/17-ga
+    releaseDate: 2021-09-15
+    eol: 2029-09-30
+    extendedSupport: 2031-09-30
+    latest: "17.40.19"
+    latestJdkVersion: "17.0.6+10"
+    latestReleaseDate: 2023-01-17
+    link: https://docs.azul.com/core/zulu-openjdk/release-notes/january-2023
+
+-   releaseCycle: "16"
+    releaseLabel: "16 (<abbr title='Short Term Support'>STS</abbr>)"
+    lts: false
+    # https://docs.azul.com/core/zulu-release-notes/16-ga/ZuluReleaseNotes/Title.htm
+    releaseDate: 2021-03-16
+    eol: 2021-09-30
+    extendedSupport: 2022-03-31
+    latest: "16.32.15"
+    latestJdkVersion: "16.0.2+7"
+    latestReleaseDate: 2021-07-24
+    link: https://docs.azul.com/core/zulu-openjdk/release-notes/july-2021
+
+-   releaseCycle: "15"
+    releaseLabel: "15 (<abbr title='Medium Term Support'>MTS</abbr>)"
+    lts: false
+    releaseDate: 2020-09-15
+    eol: 2023-03-31
+    extendedSupport: 2024-03-31
+    latest: "15.46.17"
+    latestJdkVersion: "15.0.10+5"
+    latestReleaseDate: 2023-01-17
+    link: https://docs.azul.com/core/zulu-openjdk/release-notes/january-2023
+
+-   releaseCycle: "14"
+    releaseLabel: "14 (<abbr title='Short Term Support'>STS</abbr>)"
+    lts: false
+    releaseDate: 2020-03-17
+    eol: 2020-09-30
+    extendedSupport: 2021-03-31
+    latest: "14.29.23"
+    latestJdkVersion: "14.0.2+12"
+    latestReleaseDate: 2020-07-17
+
+-   releaseCycle: "13"
+    releaseLabel: "15 (<abbr title='Medium Term Support'>MTS</abbr>)"
+    lts: false
+    releaseDate: 2019-09-17
+    eol: 2023-03-31
+    extendedSupport: 2024-03-31
+    latest: "13.54.17"
+    latestJdkVersion: "13.0.14+5"
+    latestReleaseDate: 2023-01-17
+    link: https://docs.azul.com/core/zulu-openjdk/release-notes/january-2023
+
+-   releaseCycle: "12"
+    releaseLabel: "12 (<abbr title='Short Term Support'>STS</abbr>)"
+    lts: false
+    releaseDate: 2019-03-19
+    eol: 2019-09-30
+    extendedSupport: 2020-03-31
+    latest: "12.3.11"
+    latestJdkVersion: "12.0.2+3"
+    latestReleaseDate: 2019-07-16
+
+-   releaseCycle: "11"
+    lts: true
+    releaseDate: 2018-09-25
+    eol: 2026-09-30
+    extendedSupport: 2028-09-30
+    latest: "11.62.17"
+    latestJdkVersion: "11.0.18+10"
+    latestReleaseDate: 2019-07-16
+    link: https://docs.azul.com/core/zulu-openjdk/release-notes/january-2023
+
+-   releaseCycle: "10"
+    releaseLabel: "10 (<abbr title='Short Term Support'>STS</abbr>)"
+    lts: false
+    releaseDate: 2018-03-20
```

Comment:
Best I could find was [this PR release](https://www.globenewswire.com/en/news-release/2018/03/23/1449631/0/en/Azul-Systems-Releases-Zulu-Builds-of-OpenJDK-with-Full-Support-for-Java-SE-10.html) which dates it at 23rd March, and [this release blog post](https://www.azul.com/blog/zulu-10-has-landed/) which says 27th March.

The build itself is dated 15th March:

```
curl -I https://cdn.azul.com/zulu/bin/zulu10.1+11-jdk10.0.0-linux_x64.tar.gz
...
last-modified: Thu, 15 Mar 2018 00:42:00 GMT
```

```suggestion
    releaseDate: 2018-03-27
```